core/src/apps/solana/layout.py

```diff
@@ -324,7 +332,7 @@ async def confirm_token_transfer(
     await confirm_custom_transaction(
         amount,
         decimals,
-        "[TOKEN]",
+        token_ticker,
```

Comment:
The UI question I mention above would also apply in the `confirm_value` for token mint above although here it'd be very simple to implement thanks to the `info_items` parameter.